clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -309,6 +309,7 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
                   loadBalancingPolicy.maxRatePerInstance : null,
               maxUtilization: balancingMode == GoogleHttpLoadBalancingPolicy.BalancingMode.UTILIZATION ?
                   loadBalancingPolicy.maxUtilization : null,
+              capacityScaler: loadBalancingPolicy.capacityScaler ?: 1.0,
```

Comment:
I believe this will prevent a caller from passing `0.0` for `capacityScaler`. Probably want `loadBalancingPolicy.capacityScaler != null` instead of just `loadBalancingPolicy.capacityScaler`.